app/src/main/java/com/kms/cura/model/DegreeModel.java

```diff
@@ -0,0 +1,78 @@
+package com.kms.cura.model;
+
+import com.android.volley.toolbox.StringRequest;
+import com.google.gson.Gson;
+import com.kms.cura.constant.DataConstant;
+import com.kms.cura.constant.EventConstant;
+import com.kms.cura.entity.DegreeEntity;
+import com.kms.cura.entity.SpecialityEntity;
+import com.kms.cura.entity.user.UserEntity;
+import com.kms.cura.event.EventBroker;
+import com.kms.cura.event.EventHandler;
+import com.kms.cura.model.request.DegreeModelResponse;
+import com.kms.cura.model.request.SpecialityModelResponse;
+import com.kms.cura.utils.RequestUtils;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by linhtnvo on 6/16/2016.
+ */
+public class DegreeModel implements EventHandler{
+    private static DegreeModel instance;
+    private static String tag_string_req = "string_req";
+    private DegreeModel() {
+    }
+
+    public static DegreeModel getInstace() {
+        if (instance == null) {
+            instance = new DegreeModel();
+        }
+        return instance;
+    }
+    public void getAllDegree(){
+        StringBuilder builder = new StringBuilder();
+        builder.append(Settings.SERVER_URL);
+        builder.append(Settings.GET_ALL_DEGREE);
+        EventBroker.getInstance().register(this, EventConstant.GET_ALL_DEGREE_RESPONSE_SUCCESS);
+        StringRequest stringRequest = RequestUtils.createRequestGET(builder.toString(),new DegreeModelResponse());
+        VolleyHelper.getInstance().addToRequestQueue(stringRequest,tag_string_req);
+    }
+    @Override
+    public void handleEvent(String event, String data) {
```

Comment:
this method should be DegreeModelResponse instead, to avoid sending unnecessary event